examples/applications/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# CMakeLists.txt
+
+cmake_minimum_required(VERSION 3.26)
+project(RISCVTests)
+
+set(COMPILE_FLAGS "-fsycl -Wall ${WIN_FLAG}")
+set(LINK_FLAGS "-fsycl")
+
+function(run_riscv_example_with_dpcpp)
+# To compile in a single command:
+#    clang++ -fsycl <file>.cpp -o <file>.fpga_emu
+# CMake executes:
+#    [compile] clang++ -fsycl -o <file>.cpp.o -c <file>.cpp
+#    [link]    clang++ -fsycl <file>.cpp.o -o <file>.fpga_emu
+  message("C++ Compiler: ${CMAKE_CXX_COMPILER}")
+  add_executable(${TARGET} ${SOURCES})
+  set_target_properties(${TARGET} PROPERTIES COMPILE_FLAGS "${COMPILE_FLAGS}")
+  set_target_properties(${TARGET} PROPERTIES LINK_FLAGS "${LINK_FLAGS}")
+ # add_custom_target(cpu-gpu DEPENDS ${TARGET})
```

Comment:
what's this line for?

Reply:
I missed it, it was probably from older CMakeLists. I'll remove it.